### ISSUE TITLE
 (Feature Request) Update image stylization example with EfficientNet

### ISSUE THREAD

**Issue by OleksandrGrument** [magenta/magenta#2012]
Hello,
EfficientNet is new mobile neural network architectures that is prety fast 
https://blog.tensorflow.org/2020/03/higher-accuracy-on-vision-models-with-efficientnet-lite.html

It would be great to have an example of training with this network architecture insted of VGG-16
https://github.com/magenta/magenta/tree/main/magenta/models/image_stylization
